shared/src/business/entities/cases/PublicCase.ts

```diff
@@ -50,11 +49,7 @@ PublicCase.prototype.init = function init(rawCase, { applicationContext }) {
 
   const currentUser = applicationContext.getCurrentUser();
 
-  if (
-    (currentUser.role === ROLES.irsPractitioner ||
-      currentUser.role === ROLES.privatePractitioner) &&
-    !this.isSealed
-  ) {
+  if (currentUser.role === ROLES.irsPractitioner && !this.isSealed) {
```

Comment:
Hey, what is the reason behind no longer allowing private practitioners to see parties on a case?

Reply:
Prior to Story 9874, a private practitioner if they were not party to that case, they got the public contact for the entity name and therefore only could see the petitioner name, contactType and contactId.  We are simply reverting it back to that if they are not the practitioner of record on that case as per story 10034.

Reply:
ok, thanks for explaining that to me!

Reply:
sorry you will have some conflicts on this file because Rosie and I changed it to be a class to match the other TS entities and our code got merged to test recently!

Reply:
Ur welcome